app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -111,11 +111,21 @@ class TransactionListFragment : BaseViewBindingFragment<FragmentTransactionListB
                 override fun getItemOffsets(outRect: Rect, view: View, parent: RecyclerView, state: RecyclerView.State) {
                     super.getItemOffsets(outRect, view, parent, state)
                     when (val viewHolder = nullOnThrow { parent.getChildViewHolder(view) }) {
-                        is ConflictViewHolder -> if (viewHolder.hasNext()) return
+                        is ConflictViewHolder -> {
+                            if (viewHolder.hasNext()) {
+                                return
+                            } else {
+                                addItemSeparator(outRect)
+                            }
+                        }
                         is SectionConflictHeaderViewHolder -> return
-                        else -> outRect[0, 0, 0] = context.resources.getDimension(R.dimen.item_separator_height).toInt()
+                        else -> addItemSeparator(outRect)
                     }
                 }
+
+                private fun addItemSeparator(outRect: Rect) {
+                    outRect[0, 0, 0] = context.resources.getDimension(R.dimen.item_separator_height).toInt()
```

Comment:
can we access resources directly?

Reply:
63d7f80ef0b8dc000158d87a506fab2954bc589a